frontend/src/component/environments/EnvironmentTable/EnvironmentActionCell/EnvironmentCloneModal/EnvironmentCloneModal.tsx

```diff
@@ -184,7 +184,7 @@ export const EnvironmentCloneModal = ({
                 newToken(token);
             }
             setToastData({
-                title: 'Environment successfully cloned!',
+                text: 'Environment successfully cloned!',
```

Comment:
probably we can drop the exclamation mark. We don't get excited about other commands so probably no reason to get excited about cloning :D

Reply:
I mean, there's lots of reasons to get excited about cloning, but maybe not so much about cloning envs 😄